packages/shared/src/modules/users/libs/types/user-info-response-dto.type.ts

```diff
@@ -0,0 +1,6 @@
+type UserInfoResponseDto = {
+	email: string;
```

Comment:
we need to return id for entities
```suggestion
type UserPatchResponseDto = {
    id: number;
	email: string;
```